sys/mobile_proxy.js

```diff
@@ -45,12 +136,18 @@ class MobileApps {
             // new renders are done from scratch, so comparing makes no sence as
             // templates might be dynamic.
             if (mwUtils.isNoCacheRequest(req) && oldBucket && newBucket) {
-                if (JSON.stringify(oldBucket.body) !== JSON.stringify(newBucket.body)) {
-                    hyper.log('error/mobileapps', {
-                        message: 'Content mismatch between old and new bucket',
-                        old_etag: oldBucket.headers.etag,
-                        new_etag: newBucket.headers.etag
-                    });
+                const oldBody = oldBucket.body;
+                const newBody = newBucket.body;
+                if (stringify(oldBody) !== stringify(newBody)) {
+                    const diff = this._getDiff(oldBody, newBody);
+                    if (diff.length) {
+                        hyper.log('error/mobileapps/mismatch', {
+                            message: 'Content mismatch between old and new bucket',
+                            old_etag: oldBucket.headers.etag,
+                            new_etag: newBucket.headers.etag,
+                            diff
```

Comment:
Let's stringily the diff so that log stash doesn't try to analyze all of it's properties - we might hit into the problem with field types.